### PR TITLE
Fix/gui mounting rotation

### PR DESCRIPTION
**Pull request by loucass003**

### CODE REVIEW

**Comment by loucass003** [comment]
Fix https://github.com/SlimeVR/SlimeVR-Server/issues/332

**Review by ButterscotchV — APPROVED** [review]
Looks good and seems to work 👍

**Comment by loucass003** [comment]
Original front
mountingRotation:
  x: 0.0
  "y": 1.0
  z: 0.0
  w: 6.123234E-17

New front
mountingRotation:
  x: 1.0
  "y": 0.0
  z: 0.0
  w: 6.123234E-17
  
  Not setting the right axis, need fixing

File: gui/src/components/tracker/TrackerSettings.tsx [hunk 1]
```diff
@@ -59,8 +59,13 @@ export function TrackerSettingsPage() {
...
 
     const assignreq = new AssignTrackerRequestT();
+
     assignreq.mountingRotation = QuaternionToQuatT(
-      Quaternion.fromEuler(0, +mountingOrientation, 0)
+      Quaternion.fromEuler(
+        0,
+        FixEuler(+mountingOrientation) * (Math.PI / 180),
+        0
+      )
```
Comment (ButterscotchV) [inline gui/src/components/tracker/TrackerSettings.tsx:68]: According to all the documentation, this *should* be the right order to apply and everything
```suggestion
      Quaternion.fromEuler(
        0,
        0,
        FixEuler(+mountingOrientation) * (Math.PI / 180),
        'XZY'
      )
```

**Review by ButterscotchV — CHANGES_REQUESTED** [review]
Needs to have the Quaternions fixed before it can be merged

**Review by ButterscotchV — APPROVED** [review]

**Pull request merged**